rqt_py_common/src/rqt_py_common/topic_helpers.py

```diff
@@ -28,161 +28,243 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-import roslib
-import roslib.msgs
-import roslib.message
-from rostopic import get_topic_type
-from python_qt_binding.QtCore import qDebug
+import rclpy
```

Comment:
These imports should be in alphabetical order

Reply:
Seems low priority but I did it anyway